.travis.yml

```diff
@@ -1,6 +1,16 @@
 language: java
+sudo: false
+install: true
```

Comment:
@thboileau Might be related indeed. You can try w/o `sudo:false` to see if it works - this should not have too many impacts on SQ analysis. Maybe another solution is to increase the heap size when executing the unit tests (like mentioned [here](https://github.com/travis-ci/travis-ci/issues/3396#issuecomment-92783270)).